cli/command/image/build/context_test.go

```diff
@@ -245,6 +245,8 @@ func TestValidateContextDirectoryWithOneFileExcludes(t *testing.T) {
 func createTestTempDir(t *testing.T, prefix string) (string, func()) {
 	path, err := ioutil.TempDir("", prefix)
 	assert.NilError(t, err)
+	path, err = filepath.EvalSymlinks(path)
+	assert.NilError(t, err)
```

Comment:
Thanks for your review @thaJeztah, I fixed this issue to eval symlinks and now tests are ok on MacOS runners.